index.html

```diff
@@ -1,168 +1,161 @@
-            Sed eveniet molestias doloribus, incidunt explicabo nihil est voluptates ea voluptas natus quisquam et dolorum, officiis omnis temporibus excepturi accusantium. Ipsa quisquam porro ipsum placeat provident quibusdam officiis ex autem?
-            Quam reiciendis obcaecati qui, sint, sequi culpa, at assumenda voluptatem distinctio autem doloribus enim aliquid nobis earum ea perferendis pariatur? Quae itaque cumque illum maxime explicabo nam aliquam earum sapiente!
-            Qui accusantium sit nulla nobis quae laudantium tempora ad dolor culpa possimus, voluptatum sint sequi officia eum quas minima laboriosam molestiae animi rem iste. Atque quibusdam itaque praesentium possimus cum.
-            Quaerat officia corrupti ad temporibus hic quod laudantium, laboriosam molestiae ducimus sequi voluptate perferendis sint dicta minus delectus eveniet ipsam ratione totam architecto atque. Ipsum alias iste tenetur at fuga.
-            Deleniti accusamus minus aut ipsam rerum doloremque sunt vel ducimus, reprehenderit quisquam id. Non culpa quas quia necessitatibus ut vero. Quidem quibusdam doloremque voluptatibus unde distinctio similique aliquam inventore eligendi.
-            Iure rem doloremque cum assumenda dignissimos illo eum. Corporis, fuga repellat. Consequuntur vitae quos qui nesciunt itaque eius in, ullam doloremque dignissimos ipsum autem, voluptatem, consequatur neque quod facere reiciendis!
-            Tempore, accusantium, eius in itaquequos optio maiores illum perspiciatis ad eos quaerat aperiam ex suscipit, dolorum, architecto error. Quidem impedit minima dolore delectus repellat omnis hic eius enim perferendis.</p>
-                
-            <h3>My hobbies</h3>
-                <ul>
-                    <li>Travelling</li>
-                    <li>Drawing</li>
-                    <li>Hiking</li>
-                </ul>
-
-            <h3>My skills</h3>
-                <ul>
-                    <li>Shell</li>
-                    <li>Git/Github</li>
-                    <li>HTML</li>
-                </ul>
         
-        </section>
-
-        <hr>
-
-        <section id="contact">
-            <h2>Contact</h2>
-            <div>Mobile: 0176/41518761</div>
-            <div>Email: anita.s1@gmx.de</div>
-        </section>
+        <section id="aboutme" class="aboutme"> 
+
+            <h2>ABOUT ME</h2>
+
+            <section class="portrait"> 
+                <img src="images/anita-schiller.jpg" alt="Portrait Anita Schiller" class="image">
+                <figcaption>Welcome to my page!</figcaption>
+            </section>
+
+            <section id="contact" class="contact">
+                <h3>CONTACT</h3>
+                    <div class="contact-item">
+                        <span class="headline">Mobile:</span><br>
```

Comment:
If you will use the class `headline` for other headlines this name is fine. Otherwise you could be more precise in the naming to connect it with the contact area